### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by naivenom**
Payloads template info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,11 @@ $ True
 <img src="ss.png"  height="450px" width="800px" alt="xira" />
 </p>
 
+### Payloads Template
+We can use payload_template.json to use payloads from Seclists or wherever you want. final.json will be our new payload.json file for using in our XSS Scanning.
```
Comment (naivenom) [inline README.md:55]: payload_template.json file content:
```json
{
    "payload": [
        {
            "payload_name": " <h2>Basic template payloads</h2> "
        }
    ]
}
```

File: README.md [hunk 2]
```diff
@@ -51,6 +51,11 @@ $ True
 <img src="ss.png"  height="450px" width="800px" alt="xira" />
 </p>
```
Comment (xadhrit) [inline README.md:53]: Thanks to @naivenom for adding Payload Bash Generator.

**Review by naivenom — COMMENTED** [review]
Payload Bash generator, using XSS.txt files like Seclists payloads
```bash
#!/bin/bash
filename='XSS-Somdev.txt'
n=1
while read line; do
# reading each line
jq --arg value "$line" '.payload|=.+[{ "payload_name" : $value }]' payload_template.json >tmp.json
mv tmp.json payload_template.json
echo "Line No. $n : $line"
n=$((n+1))
done < $filename
```

**Comment by xadhrit** [comment]
hey @naivenom :)
check out 8IP: https://github.com/xadhrit/8IP.git. Can you help me to make this tool more cooler

**Review by xadhrit — COMMENTED** [review]
Done

**Pull request merged**